### PR TITLE
Deploy MWAA using CICD

### PR DESCRIPTION
**Pull request by amarouane-ABDELHAK**
Automated deployment of MWAA using CICD in dev environment

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -7,4 +7,6 @@ SUBNET_TAGNAME=<Fill Me>
 STATE_BUCKET_NAME=<Fill Me>
 STATE_BUCKET_KEY=<Fill Me>
 STATE_DYNAMO_TABLE=<Fill Me>
-ASSUME_ROLE_ARN=<Fill Me>
+ASSUME_ROLE_ARNS='["<Fill Me>", "<Fill Me>"]'
```
Comment (slesaad) [inline .env.example:10]:
```suggestion
ASSUME_ROLE_ARNS='["<Read role>", "<Write role>"]'
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+* Detailed list or prose of changes
+* ...
+
+## PR Checklist
+
+- [ ] Update CHANGELOG
+- [ ] Unit tests
+- [ ] Ad-hoc testing - Deploy changes and test manually
+- [ ] Integration tests
```
Comment (slesaad) [inline .github/pull_request_template.md:15]: kinda ironic that you've added this template, but the PR doesn't follow it 😂

Comment (amarouane-ABDELHAK) [reply .github/pull_request_template.md:15]: It will be effected after the merge, right now there is no PR template

**Review by moradology — APPROVED** [review]
LGTM. I'm obviously less than 100% confident about the Terraform as it is relatively difficult to spot-read but we should know soon enough through CI/CD if there are any issues

**Pull request merged**